src/components/LandingHero/index.tsx

```diff
@@ -42,28 +42,29 @@ export const LandingHero: FC = () => (
             <strong className='text-green'>
               offene Datenplattform für IoT-Projekte
             </strong>{" "}
-            bei der jede*r mitmachen kann. Einfach deine Sensoren registrieren
-            und alle Messwerte werden automatisch gespeichert, visualisiert und
-            für andere verfügbar gemacht.
+            Deiner Stadt. Und das beste: jede*r kann mitmachen. Egal ob Du
+            offene Daten suchst oder bereits eigene Sensoren und Daten hast:
+            Stadtpuls hilft dir, tiefer in die Welt der Stadt-Sensorik
+            einzutauchen und deine Stadt besser zu verstehen.
           </p>
-          <Link href='/docs'>
+          <Link href='/'>
             <AnchorButton
               href='/'
               variant='primary'
               className='animate-slide-in mr-4 mb-2'
               style={{ animationDelay: "150ms" }}
             >
-              Mehr Erfahren
+              Mehr erfahren
             </AnchorButton>
           </Link>
-          <Link href='/projects'>
+          <Link href='/docs'>
             <AnchorButton
-              href='/projects'
+              href='/docs'
               variant='secondary'
               className='animate-slide-in'
               style={{ animationDelay: "200ms" }}
             >
-              Projekte ansehen
+              Mitmachen
```

Comment:
Here "Mehr efrahren" should say "Mitmachen", lead to the docs, be primary and come last. 
The second button should remain "Projekte ansehen", be secondary, come first and lead to the projects page.

Reply:
Good idea - I updated this!